watertap/core/membrane_channel0d.py

```diff
@@ -114,21 +116,29 @@ def add_state_blocks(self, has_phase_equilibrium=None):
                     },
                 },
             )
-        else:
+        elif self._flow_direction == FlowDirection.backward:
+            # TODO: added this more explicit conditional as a sanity check,
+            # but it seems the intended referencing below does not work, i.e.,
+            # properties_in[0] == properties[0, 0] (what's actually happening)
+            # instead of properties_in[0] == properties[0, 1] (intended but not happening)
```

Comment:
@bknueven Unsure why this didn't work for the permeate channel which is setup to have backward flow direction. However, taking this code outside and putting directly in oaro_0D after `add_state_blocks` ending up creating the intended object references. For now, I have the intended functionality working, but would like you to take a look.